### ISSUE TITLE
Task-level remote input localization 

### ISSUE THREAD

**Issue by sitapriyamoorthi** [FredHutch/wdl-unit-tests#2]
1. If inputs at the task level are being correctly localized from remote (non-execution folder)  and made available to a task and are successfully copied into the execution folder
2. Also testing the use of string, integer, float and boolean as inputs to a task
3. Also test if the output from one task is properly being made available for use as an input in another task
4. If task outputs are localized to the execution directory

**Comment by sitapriyamoorthi** [comment]
While this [unit test](https://github.com/FredHutch/wdl-unit-tests/pull/25) addresses localization of file inputs we need to address what handles the scenario for files stored in fast or temp:

**Comment by sitapriyamoorthi** [comment]
In the future we should write a test to evaluate what happens if you dont have access.

**Comment by sitapriyamoorthi** [comment]
DO NOT MERGE ASSOCIATED BRANCH TO MAIN... NEEDS PROOF ACCESS

**Comment by sitapriyamoorthi** [comment]
Blocked by infrastructure
